changelogs/unreleased/libcurl-options-fix.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/build
+
+* Several forgotten curl options set to default (gh-10576).
```

Comment:
It is not part of the #10576 libcurl update in my understanding.

Reply:
Removed issue number.